lib/active_merchant/billing/gateways/place_to_pay.rb

```diff
@@ -0,0 +1,235 @@
+module ActiveMerchant #:nodoc:
+  module Billing #:nodoc:
+    class PlaceToPayGateway < Gateway
+      self.test_url = 'https://api-co-dev.placetopay.ws/'
+      self.live_url = 'https://api-co.placetopay.ws/'
+
+      self.default_currency = 'COP'
+
+      self.supported_countries = ['COL']
+      self.supported_cardtypes = %i[visa master american_express discover]
+      self.homepage_url = 'https://www.placetopay.com/'
+      self.display_name = 'PlaceToPay'
+
+      STANDARD_ERROR_CODE_MAPPING = {}
+
+      ISO_FORMAT_DATE = '%Y-%m-%dT%H:%M:%S%:z'
+
+      def initialize(options = {})      
+        requires!(options, :login, :secret_key)
+        @login, @secret_key = options.values_at(:login, :secret_key)
+        super
+      end
+
+      def purchase(money, payment, options = {})
+        post = {}
+        add_auth(post, options)
+        add_payer(post, options)
+        add_payment(post, options)
+        add_instrument(post, payment, options)
+        commit(:post, 'gateway/process', post)
+      end
+
+      def capture(money, authorization, options = {})
```

Comment:
I noticed in the email from Juliana she mentioned the gateway does not support authorize. What purpose is capture serving if it is not proceeded by authorize? Is it necessary to have as a method/transaction type for this gateway if not being used in tandem with authorize?

Reply:
Good morning! Authorize and capture methods were removed at the moment, since they are not supported. Please let me know if you have any concern

Reply:
I see in the gateway docs there is `checkin` and `checkout`. Are these `authorize` and `capture` but just called something else? Also can you clarify the difference between reverse and refund in the gateway docs? In the ActiveMerchant transaction types there is the option for a `credit` which allows a merchant to refund money/funds without a transaction reference.